Makefile

```diff
@@ -30,6 +30,7 @@ build_readme:
 	cat docs/readme/examples_source/extractor/train.md >> ${README_FILE}
 	cat docs/readme/examples_source/extractor/val.md >> ${README_FILE}
 	cat docs/readme/examples_source/extractor/train_val_pl.md >> ${README_FILE}
+	cat docs/readme/examples_source/extractor/train_2loaders_val.md >> ${README_FILE}
 	cat docs/readme/examples_source/extractor/retrieval_usage.md >> ${README_FILE}
```

Comment:
we don't add all the examples to readme, and this particular example is too specific for the main readme